task_3/PoseAndColor[] data